Sources/DeepLearning/Layer.swift

```diff
@@ -1169,6 +1169,33 @@ public struct UpSampling2D<Scalar: TensorFlowFloatingPoint>: Layer {
     }
 }
 
+/// An upsampling layer for 3-D inputs.
+@_fixed_layout
+public struct UpSampling3D<Scalar: TensorFlowFloatingPoint>: Layer {
+    @noDerivative public let size: Int
+
+    /// Creates an upsampling layer.
+    ///
+    /// - Parameter size: The upsampling factor for rows and columns.
+    public init(size: Int) {
+       self.size = size
+    }
+
+    /// Returns the output obtained from applying the layer to the given input.
+    ///
+    /// - Parameters:
```

Comment:
`Layer.swift` isn't consistent, but could you please use:
```suggestion
    /// - Parameter input: The input to the layer.
```